guidance/01-reproducible/pages/01-documented.qmd

```diff
@@ -1,4 +1,68 @@
 ---
 title: "Documented"
-description: ""
----
\ No newline at end of file
+description: "Effective documentation is crucial for the health, usability, and maintainability of any software project."
+---
+
+> ## Pre-requisite reading
+>
+> * [AF Duck book: Code documentation](https://best-practice-and-impact.github.io/qa-of-code-guidance/code_documentation.html)
+> * [AF Duck book: Project documentation](https://best-practice-and-impact.github.io/qa-of-code-guidance/project_documentation.html)
+> * [NHSBSA DDaT playbook: Licensing software or code](https://nhsbsa.github.io/nhsbsa-digital-playbook/development/coding-licences/)
+> * [NHSBSA DDaT playbook: Standard code repository files](https://nhsbsa.github.io/nhsbsa-digital-playbook/development/dev-documentation/)
+
+Effective documentation is crucial for the health, usability, and maintainability of any software project. It encompasses both the specifics within the code and the broader context of the project itself.
```

Comment:
Took a few tries to get that second sentence, difficult to capture everything succintly. I think this does capture the main types of documentation and ties back to the principles mentioned in the first sentence.

```suggestion
Effective documentation is essential for any software project to be reproducible, quality assured and transparent. It encompasses both code and the project itself, providing technical details, project management information, records of quality assurance and details of logic, assumptions and decisions made.
```